modules/installation-configuration-parameters.adoc

```diff
@@ -627,11 +627,13 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifndef::openshift-origin[]
 ifdef::aws[]
 |`platform.aws.lbType`
```

Comment:
@ShaunaDiaz Your published output looks good. However, I believe this should be `controlPlane.platform.aws.lbType`

Reply:
It is listed exactly as dev required, so let's leave it as is.